### PR TITLE
EDU-830 Restore volume with unmutting and support solo

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-830

Screen Shot 2022-09-27 at 17 46 15
Screen Shot 2022-09-27 at 17 46 28

### CODE REVIEW

File: src/components/media-player/media-volume-slider.js [hunk 1]
```diff
@@ -10,12 +10,21 @@ export const MEDIA_VOLUME_SLIDER_ORIENTATION = {
 };
 
 function MediaVolumeSlider({ value, onChange, orientation }) {
+  const [lastValueBeforeMutting, setLastValueBeforeMutting] = useState(value);
```
Comment (ahelmberger) [inline src/components/media-player/media-volume-slider.js:13]: Typo: `Muting`

**Comment by ahelmberger** [comment]
> Closes https://educandu.atlassian.net/browse/EDU-830
> 
> Screen Shot 2022-09-27 at 17 46 15
> 
> Screen Shot 2022-09-27 at 17 46 28

It would be cool if we had a speaker logo with an "S" in it (in most applications the "Mute" and "Solo" buttons are just labeled "M" and "S"). Maybe we can ask Mani to create one and exchange it then.

File: src/components/media-player/media-volume-slider.js (hunk 1 above)
Comment (adrianaluchian) [reply src/components/media-player/media-volume-slider.js]: I appreciate you calling it a typo, given that I intentionally wrote it like that thinking it is correct 😶 
Corrected, thanks!

**Comment by adrianaluchian** [comment]
> 
> It would be cool if we had a speaker logo with an "S" in it (in most applications the "Mute" and "Solo" buttons are just labeled "M" and "S"). Maybe we can ask Mani to create one and exchange it then.

After failing my attempt to create a dedicated icon, I went for the colored variation for now, but I agree we should get a proper icon. "M" would only make sense in combination with "S", but in most use cases of the volume slider there is no solo. 
"S" though makes sense on its own I'd say 👍

File: src/components/media-player/media-volume-slider.js (hunk 1 above)
Comment (ahelmberger) [reply src/components/media-player/media-volume-slider.js]: :smile:

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**